ESP32/main/src/serial/SerialProcess.cpp

```diff
@@ -0,0 +1,244 @@
+#include "SerialProcess.h"
+
+SerialProcess::SerialProcess(/* args */)
+{
+}
+
+SerialProcess::~SerialProcess()
+{
+}
+
+void SerialProcess::loop(DynamicJsonDocument * jsonDocument)
+{
+#ifdef IS_SERIAL
+    config.loop(); // make it sense to call this everyime?
+    if (Serial.available())
+    {
+        DeserializationError error = deserializeJson((*jsonDocument), Serial);
+        // free(Serial);
+        if (error)
+        {
+            Serial.print(F("deserializeJson() failed: "));
+            Serial.println(error.f_str());
+            return;
+        }
+        Serial.flush();
+#ifdef DEBUG_MAIN
```

Comment:
for now DEBUG_MAIN can fully disable serial parsing. 
recommend not to use Serial.print/ln() for sending logs.
Esp have there ESP_LOGI from espressif. it also dont mix up the logoutput as it could happen with serial.print.

Lets try to overwhelm the logcat ;)